resources/api-docs/1.0/attributes.md

```diff
@@ -97,6 +99,7 @@ To create an attribute, use the following endpoint.
 | `entity_id` | `int` (Required) | The attribute's parent entity |
 | `is_private` | `boolean` | If the attribute is only visible to `admin` members of the campaign |
 | `is_star` | `boolean` | If the attribute is "pinned" on the entity view |
```

Comment:
We can remove `is_star` from the docs, because we want people to use `is_pinned` instead